dissect/target/helpers/configutil.py

```diff
@@ -59,39 +102,70 @@ def keys(self) -> KeysView:
     def items(self) -> ItemsView:
         return self.parsed_data.items()
 
-    def _collapse_dict(self, dictionary: dict, collapse: bool = False) -> dict[str, dict]:
-        new_dictionary = {}
 
-        if isinstance(dictionary, list) and collapse:
-            return dictionary[-1]
+class Default(ConfigurationParser):
+    """Parse a configuration file specified by ``seperator`` and ``comment_prefixes``.
 
-        if not hasattr(dictionary, "items"):
-            return dictionary
+    This parser splits only on the first ``seperator`` it finds:
 
-        for key, value in dictionary.items():
-            value = self._collapse_dict(value, self.collapse_all or key in self.collapse)
-            new_dictionary.update({key: value})
+        key<seperator>value    -> {"key": "value"}
 
-        return new_dictionary
+        key<seperator>value\n  -> {"key": "value continuation"}
+          continuation
```

Comment:
```suggestion
        key<seperator>value\n
          continuation
                                 -> {"key": "value continuation"}
```

Would formatting it like this be more clear? (At least make the 3rd one consistent with the first 2 multi-line parse examples).